src/apps/umami/umami.definition.ts

```diff
@@ -5,7 +5,7 @@ import { Network } from '~types/network.interface';
 
 export const UMAMI_DEFINITION = appDefinition({
   id: 'umami',
-  name: 'umami',
+  name: 'UMAMI',
```

Comment:
Perhaps `Umami Finance` like described on the Twitter account? https://twitter.com/UmamiFinance

Reply:
<img width="238" alt="Capture d’écran, le 2022-06-21 à 11 07 43" src="https://user-images.githubusercontent.com/6602520/174868840-332ff842-c54f-4834-b2b3-68e91ad10ed7.png">

Regarding this issue, I changed it to match the token's name so the image would be displayed for both the token and the app. 

Is there another way to do so ? If so I'll change the name for sure.

Reply:
And the name can be anything friendly to the user, so I suggest `Umami Finance`